homeassistant/components/aladdin_connect/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Config flow for Aladdin Connect cover integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aladdin_connect import AladdinConnectClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("username"): str,
+        vol.Required("password"): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> None:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+    acc = AladdinConnectClient(data["username"], data["password"])
+    try:
+        login = await hass.async_add_executor_job(acc.login)
+    except (TypeError, KeyError, NameError, ValueError) as ex:
+        raise CannotConnect from ex
+    else:
+        if not login:
+            raise InvalidAuth
+
+    return
```

Comment:
```suggestion
```
You can remove this since it is not required when not expecting to return anything